package.json

```diff
@@ -45,9 +49,12 @@
   },
   "homepage": "https://github.com/ProtonMail/pmcrypto#readme",
   "dependencies": {
+    "@openpgp/asmcrypto.js": "^2.3.2",
+    "@openpgp/tweetnacl": "^1.0.3",
+    "@openpgp/web-stream-tools": "0.0.6",
     "@types/openpgp": "^4.4.12",
     "esm": "^3.0.84",
-    "openpgp": "^4.10.8"
+    "openpgp": "5.0.0"
```

Comment:
`~5.0.0` ?

Reply:
Hmhm, or "^5.0.0" to be consistent with the others :shrug: